content/exporters/supported-exporters/Python/Honeycomb.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Honeycomb.io (Tracing)"
+date: 2010-03-18
+draft: false
+weight: 3
+class: "resized-logo"
+aliases: [/guides/exporters/supported-exporters/python/honeycomb]
+logo: /img/honeycomb-logo.jpg
+---
+
+- [Introduction](#introduction)
```

Comment:
Below you'll see a request to add some other sections
\- \[Installing it\](#installing-it)